.github/workflows/build_llvm.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  push:
+    branches: [ release_90 ]
+    
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        target: [X86, AArch64, PowerPC]
+
+    steps:
+      - name: Check tools
+        run: |
+          pwd
+          git --version
+          cmake --version
+          make --version
+          gcc --version
+          
+      - name: Build llvm
+        run: |
+          # exit the llvm dir, so the path is the same on subsequent projects
+          cd ../..
+          rm -rf llvm
+          pwd
+          # clone manually, because checkout does not allow exiting llvm dir
+          git clone --depth 1 --single-branch --branch release_90 https://github.com/flang-compiler/llvm.git
+          cd llvm
+          pwd
+          CMAKE_OPTIONS="-DLLVM_TARGETS_TO_BUILD=${{ matrix.target }}"
+          mkdir -p build && cd build
+          cmake $CMAKE_OPTIONS -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER=/usr/bin/gcc-9 -DCMAKE_CXX_COMPILER=/usr/bin/g++-9 ..
+          make -j$(nproc)
+          # Install so the flang-driver can be built
+          sudo make install
```

Comment:
What is the  installation path? It's not clear since `CMAKE_INSTALL_PREFIX` is not set.

Reply:
This was also a point raised by @banach-space.

The default installation path on Ubuntu is `/usr/local`, but actually I didn't care much as long as it is the same for all cmake scripts because our workspace is temporary and gets destroyed after every job run. 

You're right - we could explicitly set it with the macro, but this would require repeating the set up for every command and would make the scripts less readable and longer. Every `step` runs in a new shell and has to set up `CMAKE_OPTIONS` anew.

I think defaults are there for us to use them and they are set up in a reasonable and consequent way. So if we don't care about the path in this case, I though it's best to keep the scripts simple...

Reply:
I'm always in favor of being explicit with these things.

Today the installation path seems obvious as we've discussed it recently. However, we also want to make sure that:
* it is obvious to us in not-so-distant future
* it is obvious to anyone who hasn't participated in this PR

In the current set-up we rely on people knowing what the default is. By setting `CMAKE_INSTALL_PREFIX` we no longer rely on that knowledge. Instead, we define and document what that path is. So yes, my preference would be to set `CMAKE_INSTALL_PREFIX`. Even if initially it's set to `/usr/local`.

@michalpasztamobica Thank you for working on this!

Reply:
@bryanpkc, @banach-space, thanks for you remarks!

I just pushed an update to all three projects including your suggestion to be explicit about the path and also some more slight improvements:
* I added a scheduled build once every second month - this is to make sure we have artifacts in place even if there would be no pushes to the repo for more than 90 days (unlikely, but better be sure)
* I packaged the build directory along with source code, to avoid the situation where versions don't match (thanks for the suggestion, @bryankpc)

The flow is still working, see [llvm build](https://github.com/michalpasztamobica/llvm/actions/runs/337486934), [flang-driver build](https://github.com/michalpasztamobica/flang-driver/actions/runs/337592116) and [flang build](https://github.com/michalpasztamobica/flang/actions/runs/337689520) .

I placed changes in a separate commit to make the review easier. I'll squash and rename the project to `flang-compiler` once the review is approved.